Required Configuration Blocks Not Providing Which Block Is Missing in Error

### SDK version

```
v2.0.1
```

### Relevant provider source code

```go
func resourceAwsEksNodeGroup() *schema.Resource {
	return &schema.Resource{
		/* ... */
		Schema: map[string]*schema.Schema{
			/* ... */
			"scaling_config": {
				Type:     schema.TypeList,
				Required: true,
				MaxItems: 1,
				Elem: &schema.Resource{
					Schema: map[string]*schema.Schema{
						"desired_size": {
							Type:         schema.TypeInt,
							Required:     true,
							ValidateFunc: validation.IntAtLeast(1),
						},
						"max_size": {
							Type:         schema.TypeInt,
							Required:     true,
							ValidateFunc: validation.IntAtLeast(1),
						},
						"min_size": {
							Type:         schema.TypeInt,
							Required:     true,
							ValidateFunc: validation.IntAtLeast(1),
						},
					},
				},
			},
		},
	}
}
```

### Terraform Configuration Files

```hcl
resource "aws_eks_node_group" "test" {
  # ... other configuration except scaling_config ...
}
```

### Debug Output

Please ask if you need this.

### Expected Behavior

The acceptance testing framework should output which attribute name is missing, when reporting the `Required attribute is not set` error.

### Actual Behavior

```
=== CONT  TestAccAWSEksNodeGroup_LaunchTemplate_Id
    TestAccAWSEksNodeGroup_LaunchTemplate_Id: resource_aws_eks_node_group_test.go:312: Step 1/3 error: terraform failed: exit status 1

        stderr:

        Error: Required attribute is not set

          on ../../../../../../../../var/folders/w8/05f3x02n27x72g0mc2jy6_180000gp/T/tftest038613874/work421440041/config093175149/terraform_plugin_test.tf line 171, in resource "aws_eks_node_group" "test":
         171: resource "aws_eks_node_group" "test" {


--- FAIL: TestAccAWSEksNodeGroup_LaunchTemplate_Id (2.09s)
```

### Steps to Reproduce

1. Find a resource with a `Required: true` attribute
1. Find a test for that resource and remove the required attribute from the configuration
1. Run acceptance test

--- a/helper/schema/schema.go
+++ b/helper/schema/schema.go
@@ -1447,7 +1447,8 @@ func (m schemaMap) validate(
 		if schema.Required {
 			return append(diags, diag.Diagnostic{
 				Severity:      diag.Error,
-				Summary:       "Required attribute is not set",
+				Summary:       "Missing required argument",
+				Detail:        fmt.Sprintf("The argument %q is required, but no definition was found.", k),
 				AttributePath: path,
 			})
 		}
@@ -1458,7 +1459,8 @@ func (m schemaMap) validate(
 		// This is a computed-only field
 		return append(diags, diag.Diagnostic{
 			Severity:      diag.Error,
-			Summary:       "Computed attribute cannot be set",
+			Summary:       "Computed attributes cannot be set",
+			Detail:        fmt.Sprintf("Computed attributes cannot be set, but a value was set for %q.", k),
 			AttributePath: path,
 		})
 	}

